internal/runbits/runtime/requirements/requirements.go

```diff
@@ -246,7 +246,7 @@ func (r *RequirementOperation) ExecuteRequirementOperation(ts *time.Time, requir
 
 	// Report CVEs
 	names := requirementNames(requirements...)
-	if err := cves.NewCveReport(r.prime).Report(commit.BuildPlan(), oldCommit.BuildPlan(), names...); err != nil {
+	if err := cves.NewCveReport(r.prime).Report(commit, oldCommit, names...); err != nil {
```

Comment:
Is there any way we can compute the `names` here inside `Report()` and remove these extra arguments? Maybe diffing the buildscripts will be enough?

It's unclear to me when we need to pass names, and when we do not. Currently only requirements passes names, and when other commands start calling this function, are they supposed to pass names or omit them? There's an ambiguity here and I'd either like to remove it, or add a documentation comment to `Report()` as to when names are required an when they can be omitted.

Reply:
Now that we are using the build plan to get the requested requirements we should be okay to remove the `names` parameter.